makim/__main__.py

```diff
@@ -93,14 +92,12 @@ def _get_args():
         makim_file = makim_file_default
 
     makim.load(makim_file)
-
     target_help = []
-
-    for group in makim.config_data['groups']:
-        for target_name, target_data in group['targets'].items():
-            target_name_qualified = f"{group['name']}.{target_name}"
+    groups = makim.config_data['groups']
```

Comment:
```suggestion
    groups = makim.config_data.get('groups')
```
or maybe `groups = makim.config_data.get('groups', [])`
in a near future, the groups would optional .. so it would be already prepared for that.

Reply:
![image](https://github.com/osl-incubator/makim/assets/27688744/d64ca3b9-a49e-4db0-928e-835625e07028)
I can only see a similar structure in sugar.py